src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/config.scala

```diff
@@ -72,9 +73,22 @@ object config {
         bad,
         args.optional("pii"),
         resolver,
-        args.optional("enrichments")
+        args.optional("enrichments"),
```

Comment:
Shouldn't these new fields also be added to `ParsedEnrichConfig` and validated in `Enrich`?